wadas/ui/mainwindow.py

```diff
@@ -472,25 +474,49 @@ def update_en_camera_list(self):
 
     def update_en_actuator_list(self):
         """Method to list enabled actuator(s) in UI"""
-
+        threshold_time = 30
```

Comment:
This threshold could be a parameter to add in configuration so that user can tuned it... but it should be a property of the actuator or actuator server class. I would say more on actuator server to establish a single timeout for all actuators.